python_modules/dagster/dagster/_core/storage/event_log/sql_event_log.py

```diff
@@ -515,6 +517,7 @@ def get_records_for_run(
         if cursor is not None:
             cursor_obj = EventLogCursor.parse(cursor)
             if cursor_obj.is_offset_cursor():
+                deprecation_warning("Support for offset-based cursors", "1.8.0")
```

Comment:
Can we make these messages more clear? Does our own code trigger these warnings, or only users who are directly calling these from their code?

Reply:
There are no callers inside of dagster code for any of these deprecated behaviors.

Reply:
Got it. Can we add some additional warn text that provides some insight and the suggested replace code for this? "We are dropping support for offset-based cursors and will only support pagination via storage id, which provides better consistency guaranatees. Please use `EventLogCursor.from_storage_id` to construct cursors"

Or something to that effect.